indico/modules/events/operations.py

```diff
@@ -230,7 +225,7 @@ def clone_into_event(source_event, target_event, cloners):
     return target_event
 
 
-def _log_event_update(event, changes, visible_person_link_changes=False):
+def _log_event_update(event, changes, _extra_log_fields, visible_person_link_changes=False):
```

Comment:
In here I would just call it `extra_log_fields` because it's an internal function anyway - the point of marking it as internal on the `update_*` functions is to make it clear that it usually shouldn't be used (for example a developer on the core should not use it but rather update the `log_fields` dict here)